### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by Mailolipop** [TortugaPower/BookPlayer#864]
Hello BookPlayer Crew,
Can you add the option that instead of the chapter name it shows the chapter number on the lockscreen 
Instead of
![image](https://user-images.githubusercontent.com/98764652/210385177-f0759444-2a27-4cb7-b185-48ccaf317a9f.jpeg)
It will look like
![image](https://user-images.githubusercontent.com/98764652/210385502-4f060e5f-0494-4b50-bf8e-a80de5e95f9c.jpeg)

**Comment by GianniCarlo** [comment]
This could easily be a setting in the Player Controls section 👍

**Comment by Mailolipop** [comment]
I meant in the homescreen

**Comment by GianniCarlo** [comment]
right, but I mean it would be controlled by a setting, for those that prefer either this format or the current one

**Comment by maxxxxxdlp** [comment]
Maybe unrelated, but can we have a way to disable the progress bar or make it non-interactive? I often accidentally change my position in the book when getting my phone out of my pocket because of this. I never actually found the progress bar being interactive useful - for quick rewinds the buttons suffice, for longer rewinds, unlocking the phone is not that hard.

I can open a separate issue for that if you wish.

**Comment by ashabooga** [comment]
Hi there - I noticed that this issue is still unsolved and would love to take a crack at it!

**Comment by GianniCarlo** [comment]
@ashabooga that would be great! just a heads up that the player controls screen is still created from a storyboard, let me know if you have any questions. I'm gearing up for a release, I just have to do some more testing, I've done some changes in how the app loads, and how the app process chapters
